buggy_calculator.py

```
@@ -0,0 +1,33 @@
+        self.history = []
+    
+    def add(self, a, b):
+        result = a + b
+        # Bug fixed: store operation in history
+        self.history.append(f"{a} + {b} = {result}")
+        return result
+    
+    def divide(self, a, b):
+        # Bug fixed: check for division by zero
+        if b == 0:
+            raise ValueError("Cannot divide by zero")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)